docs/zh/latest/admin-api.md

```diff
@@ -821,32 +821,32 @@ Upstream 资源请求地址：/apisix/admin/upstreams/{id}
-| tls.client_cert    | 否，不能和 `tls.client_cert_id` 一起使用 | https 证书           | 设置跟上游通信时的客户端证书，详细信息请参考下文。                                                                        | |
-| tls.client_key	 | 否，不能和 `tls.client_cert_id` 一起使用 | https 证书私钥           | 设置跟上游通信时的客户端私钥，详细信息请参考下文。                                                                                                                                                                                                                                                                                                              | |
-| tls.client_cert_id | 否，不能和 `tls.client_cert`、`tls.client_key` 一起使用  | SSL           | 设置引用的 SSL id，详见 [SSL](#ssl)。                                                                                                                                                                                                                                                                                                              | |
-|keepalive_pool.size  | 否 | 辅助 | 动态设置 `keepalive` 指令，详细信息请参考下文。 |
-|keepalive_pool.idle_timeout  | 否 | 辅助 | 动态设置 `keepalive_timeout` 指令，详细信息请参考下文。 |
-|keepalive_pool.requests  | 否 | 辅助 | 动态设置 `keepalive_requests` 指令，详细信息请参考下文。 |
+| 名称           | 必选项                                           | 类型           | 描述                                                                                                                                                                                                                                                                                                                                                        | 示例                                             |
+| -------------- |-----------------------------------------------| -------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------ |
+| type           | 否                                             | 枚举           | 负载均衡算法，默认值是`roundrobin`。                                                                                                                                                                                                                                                                                                                                                           |                                      |     |
```

Comment:
False or Yes?

Reply:
this is false, please refer to: https://github.com/apache/apisix/pull/8180